### PR TITLE
DLS-8166: add application already registered page

### PR DESCRIPTION
**Pull request by RaminEsfandiari**
DLS-8166: Add utr

DLS-8166: Add controller test

DLS-8166: Add view test for header and subheader

DLS-8166: add link to account

DLS-8166: fix UTR cosmetic issue on verify

### CODE REVIEW

File: test/base/SpecBase.scala [hunk 1]
```diff
@@ -108,6 +109,7 @@ trait SpecBase
 
 
 
+
```
Comment (drewmiley) [inline test/base/SpecBase.scala:112]: Remove lines?

File: conf/messages.en [hunk 2]
```diff
@@ -300,3 +300,9 @@ removePackagingSiteDetails.title = Are you sure you want to remove this packagin
 removePackagingSiteDetails.heading = Are you sure you want to remove this packaging site?
 removePackagingSiteDetails.error.required = Select yes if you want to remove this packaging site
 removePackagingSiteDetails.change.hidden = Confirm remove this packaging site
+
+alreadyRegistered.title = You are already registered for the Soft Drinks Industry Levy
+alreadyRegistered.heading = You are already registered for the Soft Drinks Industry Levy
+alreadyRegistered.subText = These are the details we hold for Unique Taxpayer Reference (UTR)
+alreadyRegistered.go.to.account = To view your registration details, go to
```
Comment (drewmiley) [inline conf/messages.en:307]: I'd probably call this `alreadyRegistered.account.goto`

File: app/views/VerifyView.scala.html [hunk 3]
```diff
@@ -33,8 +33,7 @@
             @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
         }
         <h1 class="govuk-heading-m">@messages("verify.heading")</h1>
-        <p class="govuk-body" id="subText">@messages("verify.subText")</p>
-        <p class="govuk-body govuk-!-font-weight-bold" id="utrField">@utr</p>
+        <p class="govuk-body" id="subText">@messages("verify.subText") </br><span class="govuk-body govuk-!-font-weight-bold" id="utrField">@utr:</span></p>
```
Comment (drewmiley) [inline app/views/VerifyView.scala.html:36]: space followed by "< /br >" looks unusual?

File: app/views/AlreadyRegisteredView.scala.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+@(utr: String, formattedAddress: HtmlContent)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("alreadyRegistered.title"))) {
+
+    <h1 class="govuk-heading-m">@messages("alreadyRegistered.heading")</h1>
+
+    <p class="govuk-body" id="subheader">@messages("alreadyRegistered.subText") </br><span class="govuk-body govuk-!-font-weight-bold" id="utrField">@utr:</span></p>
+
```
Comment (drewmiley) [inline app/views/AlreadyRegisteredView.scala.html:30]: Remove one of two spacing lines

File: test/base/SpecBase.scala (hunk 1 above)
Comment (RaminEsfandiari) [reply test/base/SpecBase.scala]: Done

**Review by drewmiley — APPROVED** [review]
LGTM

**Pull request merged**